### PR TITLE
#28518 Add unit tests for SingleIndexReviser

### LINKED ISSUE #28518: Add more unit tests for SingleIndexReviser
Hi community,

This issue is for #28487.

Aim
Add more unit tests for SingleIndexReviser to test its public functions to improve unit test coverage.

Unit test coverage can be viewed at: https://app.codecov.io/gh/apache/shardingsphere

Basic Qualifications
Java
Maven
Junit

### PR DESCRIPTION
**Pull request by anogueiraFlywire**
Fixes #28518.

Changes proposed in this pull request:
  - Add unit tests for SingleIndexReviser

---

Before committing this PR, I'm sure that I have checked the following options:
- [x] My code follows the [code of conduct](https://shardingsphere.apache.org/community/en/involved/conduct/code/) of this project.
- [x] I have self-reviewed the commit code.
- [x] I have (or in comment I request) added corresponding labels for the pull request.
- [x] I have passed maven check locally : `./mvnw clean install -B -T1C -Dmaven.javadoc.skip -Dmaven.jacoco.skip -e`.
- [ ] I have made corresponding changes to the documentation. N/A
- [ ] I have added corresponding unit tests for my changes. N/A

### CODE REVIEW

File: kernel/single/core/src/test/java/org/apache/shardingsphere/single/metadata/reviser/index/SingleIndexReviserTest.java [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+class SingleIndexReviserTest {
+    
+    @Test
+    void assertReviserReturnsRevisedIndex() {
+        IndexMetaData originalMetaData = new IndexMetaData("test_idx_tableName");
+        originalMetaData.getColumns().add("column1");
+        originalMetaData.getColumns().add("column2");
+        originalMetaData.setUnique(true);
+        SingleIndexReviser reviser = new SingleIndexReviser();
+
```
Comment (tuichenchuxin) [inline kernel/single/core/src/test/java/org/apache/shardingsphere/single/metadata/reviser/index/SingleIndexReviserTest.java:38]: please remove empty line

File: kernel/single/core/src/test/java/org/apache/shardingsphere/single/metadata/reviser/index/SingleIndexReviserTest.java [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+        
+        Optional<IndexMetaData> optionalRevised = reviser.revise("tableName", originalMetaData, null);
+        
+        assertTrue(optionalRevised.isPresent());
+        IndexMetaData actual = optionalRevised.get();
+        assertThat(originalMetaData.isUnique(), is(actual.isUnique()));
+        assertThat(originalMetaData.getColumns(), is(actual.getColumns()));
+        assertThat("test_idx", is(actual.getName()));
+    }
+
```
Comment (tuichenchuxin) [inline kernel/single/core/src/test/java/org/apache/shardingsphere/single/metadata/reviser/index/SingleIndexReviserTest.java:47]: please remove this line.

File: kernel/single/core/src/test/java/org/apache/shardingsphere/single/metadata/reviser/index/SingleIndexReviserTest.java (hunk 1 above)
Comment (anogueiraFlywire) [reply kernel/single/core/src/test/java/org/apache/shardingsphere/single/metadata/reviser/index/SingleIndexReviserTest.java]: Done, though the purpose of this empty line was to separate the "given" from the "when" in the test

File: kernel/single/core/src/test/java/org/apache/shardingsphere/single/metadata/reviser/index/SingleIndexReviserTest.java (hunk 2 above)
Comment (anogueiraFlywire) [reply kernel/single/core/src/test/java/org/apache/shardingsphere/single/metadata/reviser/index/SingleIndexReviserTest.java]: Done

File: kernel/single/core/src/test/java/org/apache/shardingsphere/single/metadata/reviser/index/SingleIndexReviserTest.java [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+    
+    @Test
+    void assertReviserReturnsRevisedIndex() {
+        IndexMetaData originalMetaData = new IndexMetaData("test_idx_tableName");
+        originalMetaData.getColumns().add("column1");
+        originalMetaData.getColumns().add("column2");
+        originalMetaData.setUnique(true);
+        SingleIndexReviser reviser = new SingleIndexReviser();
+        Optional<IndexMetaData> optionalRevised = reviser.revise("tableName", originalMetaData, null);
+
```
Comment (tuichenchuxin) [inline kernel/single/core/src/test/java/org/apache/shardingsphere/single/metadata/reviser/index/SingleIndexReviserTest.java:39]: we need to remove this empty line.

Comment (anogueiraFlywire) [reply kernel/single/core/src/test/java/org/apache/shardingsphere/single/metadata/reviser/index/SingleIndexReviserTest.java]: Really? You don't even separate the asserts? I don't think that's good for readability, but ok

**Comment by anogueiraFlywire** [comment]
Hi @tuichenchuxin 
Is this ready to be merged?

**Review by tuichenchuxin — APPROVED** [review]

**Comment by tuichenchuxin** [comment]
> Hi @tuichenchuxin Is this ready to be merged?
LGTM, merged

**Pull request merged**